.rubocop.yml

```diff
@@ -0,0 +1,10 @@
+Metrics/BlockLength:
```

Comment:
Maybe this is not needed since #547 is ready to merge too. And we're using our standard stylesheet.